src/components/FormInput/FormInput.js

```diff
@@ -0,0 +1,28 @@
+import "./FormInput.scss";
+
+const FormInput = (props) => {
```

Comment:
```suggestion
const FormInput = ({ id, label, type, value, name, placeholder = "", onChange }) => {
```
You can destructure the `props` and include default values for those which aren't required. Since you're not using typescript, you could include `Proptypes` to define the types the props should have and which ones should be required

Reply:
Done!